game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
You should use `CustomNetTables.GetTableValue('team_scores', 'limit')['value'];` instead of a event.